ppdet/modeling/ops.py

```diff
@@ -205,6 +207,111 @@ def __init__(self,
         self.nms_eta = nms_eta
         self.background_label = background_label
 
+@register
+@serializable
+class MultiClassSoftNMS(object):
+    def __init__(self,
+                 class_num=81,
+                 score_threshold=0.01,
+                 keep_top_k=300,
+                 softnms_sigma=0.4,
+                ):
+        super(MultiClassSoftNMS, self).__init__()
+        self.score_threshold = score_threshold
+        self.keep_top_k = keep_top_k
+        self.softnms_sigma = softnms_sigma
+        self.class_num = class_num
+        return
+    
+    def create_tmp_var(self, program, name, dtype, shape, lod_leval):
```

Comment:
this function is used only once? why not just inline it?

Reply:
fixed~